### ISSUE TITLE
The count of real-time thread data is replaced by LongAdder

### ISSUE THREAD

**Issue by linlinisme** [alibaba/Sentinel#744]
com.alibaba.csp.sentinel.node.StatisticNode#curThreadNum

```
    /**
     * The counter for thread count.
     */
    private AtomicInteger curThreadNum = new AtomicInteger(0);
```
实时线程数据的计数采用AtomicInteger，高并发下频繁cas会造成cpu空转影响性能，换成LongAdder可以消除影响避免性能损耗。

**Comment by linlinisme** [comment]
i am going to fixed it.

**Comment by linlinisme** [comment]
i get this issue and i will modify it

**Comment by sczyh30** [comment]
Resolved in #747
